build/includes/sdk.mk

```diff
@@ -55,17 +55,22 @@ gen-all-sdk-grpc: run-all-sdk-command
 gen-sdk-grpc: COMMAND := gen
 gen-sdk-grpc: run-sdk-command
 
-# Runs a command on all supported languages, use COMMAND variable to select which command.
+# Runs a command on all supported languages in parallel, use COMMAND variable to select which command.
```

Comment:
how about using native parallelism built into make:

```
run-all-sdk-command: run-sdk-command-go run-sdk-command-rust run-sdk-command-cpp

run-sdk-command-cpp:
  $(MAKE) run-sdk-command SDK_FOLDER=cpp

run-sdk-command-rust:
  $(MAKE) run-sdk-command SDK_FOLDER=rust

run-sdk-command-go:
  $(MAKE) run-sdk-command SDK_FOLDER=go
```

Then the user can do `make -j4 run-all-sdk-command COMMAND=gen`

Reply:
ooh that's a really interesting idea. I wonder if we could also come up with a pattern rule so we don't have to define each rule individually?

Reply:
That native way looks more natural and simple, updating this Pull Request, removing initial and next waiting for loop:
```
	for job in `jobs -p` ; \
	do \
		wait $$job || let "FAIL+=1"; \
	done ; \
```

Reply:
@jkowalski
The only thing that we don't have build.sh in each directory, that's why we have such check that file exists:
`if [ "$${d%?}" != "tool" ] &&  [ -f $$d/$(COMMAND).sh ];`